src/singleRequestProxyFactory.ts

```
@@ -8,11 +8,14 @@ export function handleEthereumSingleRequestProxyCreated(
   event: EthereumSingleRequestProxyCreated,
 ): void {
   let deployment = new SingleRequestProxyDeployment(
-    event.transaction.hash.toHexString(),
+    event.transaction.hash.toHexString() + "-" + event.logIndex.toString(),
```

Biome

[error] Template literals are preferred over string concatenation.

Unsafe fix: Use a template literal.


(lint/style/useTemplate)